jobbergate-docs/src/conf.py

```diff
@@ -30,11 +30,17 @@
 pygments_style = "rainbow_dash"
 exclude_patterns = []
 extensions = [
-    "sphinx.ext.githubpages",
+    "sphinx.ext.autodoc",
     "sphinx.ext.autosectionlabel",
+    "sphinx.ext.githubpages",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.napoleon",
     "sphinxcontrib.httpdomain",
 ]
 
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3", None,),
+}
```

Comment:
Check if `requests` and/or `httpx` can be added here, to make them easier to refer.